mandatory/2-function-creation.js

```diff
@@ -5,8 +5,14 @@ Complete the function to check if the variable `num` satisfies the following req
 - is less than or equal to 100
 Tip: use logical operators
 */
+function validate(number) {
+   if(number === "number" && number % 2 === 0 && number <= 100){
```

Comment:
I think `number === "number"` isnt quite right and it check if number variable is equal to a string "number"